Pokehangmon.js

```diff
@@ -0,0 +1,282 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Overall very impressive project. Good use of class/methods, functions were used to perform specialized tasks, and sounds and visuals added nice touch to overall theme of the app. Keep up the great work and I'm looking forward to see future projects!